src/ray/raylet/scheduling/cluster_resource_scheduler.h

```diff
@@ -458,6 +458,13 @@ class ClusterResourceScheduler : public ClusterResourceSchedulerInterface {
   std::unique_ptr<NodeResources> last_report_resources_;
   /// Function to get used object store memory.
   std::function<int64_t(void)> get_used_object_store_memory_;
+
+  // Specify predefine resources that consists of unit-size instances.
+  std::unordered_set<int64_t> predefined_unit_instance_resources_{CPU, GPU};
```

Comment:
Can we just set these defaults in `ray_config_def.h`? It's a little confusing having it defined in different places.

Reply:
I have move these defaults to `ray_config_def.h`